src/main/Command.ts

```diff
@@ -0,0 +1,290 @@
+import { spawn } from 'child_process'
+import { db } from './index'
+import { getActiveDockerImage, getAllCommandConfig, getCommandConfigId } from './db'
+import {
+  AllCommandConfigureInterface,
+  DatabaseFetching,
+  ResponeInterface,
+  ResponseStatus
+} from './interfaces'
+
+export const AllDockersImages = {
+  noDockerImage: 'Run Without Docker Image',
+  jderobotRoboticsAcademy: 'jderobot/robotics-academy',
+  jderobotRoboticsBackend: 'jderobot/robotics-backend'
+}
+
+const CONTAINER_NAME = 'robotics-academy'
```

Comment:
Can we make it more configurable by making docker tags as a parameter

Reply:
@Apoorvgarg-creator , 
Yes, this would make the user experience more customizable. We can provide a selection option alongside those choices, which allows the user to input a Docker image name as a parameter.